dvc/command/experiments.py

```diff
@@ -116,6 +118,16 @@ def _collect_names(all_experiments, **kwargs):
     return metric_names, param_names
 
 
+experiment_types = {
```

Comment:
@pmrowla, let me know if you have better naming suggestions. Feels like I am artificially making one here.

Reply:
These names are fine w/me but maybe something like this would also work

```
    "branch_tip": "│ ╓",
    "branch_commit": "│ ╟",
    "branch_base": "├─╨", # or maybe branch_start
    "single_commit": "├──",
    "baseline": "",

Reply:
`branch` might be confusing with the git terminology. I instead added it to the non-checkpoint based commits.
Let me know what you think.

Reply:
the thing is that experiments really are just branches, but what you have now is fine with me

Reply:
I meant that they themselves are the branches of the commit-graph/tree, and this UI representation is all about showing those branches, no?

If you are fine with naming, please do a review. Thanks.